web-console/src/lib/components/common/table/EntityTable.tsx

```diff
@@ -64,7 +65,7 @@ const EntityTable = <TData extends GridValidRowModel>(props: EntityTableProps<TD
 
   const { isLoading, isError, data, error } = fetchRows
 
-  if (addActions) {
+  if (addActions && tableProps.columns.at(-1)?.field !== 'actions') {
```

Comment:
do we reject tableProps which are empty lists?

Reply:
No, the condition holds true if list is empty or if the last column isn't 'actions'